### PR TITLE
[Metricbeat] Add bucket name to aws.s3.bucket.name

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
Right now with two s3 metricsets, there is no common field to display bucket name. This PR is to add bucket name into `aws.s3.bucket.name` for both metricsets.

### CODE REVIEW

File: x-pack/metricbeat/module/aws/s3_daily_storage/_meta/data.json [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
     "@timestamp": "2017-10-12T08:05:34.853Z",
     "aws": {
+        "s3": {
+            "bucket": {
+                "name": "test-s3-ks-2"
+            }
+        },
         "s3_daily_storage": {
             "bucket": {
                 "name": "test-s3-ks-2",
```
Comment (jsoriano) [inline x-pack/metricbeat/module/aws/s3_daily_storage/_meta/data.json]: Should we mark this field as deprecated?

File: x-pack/metricbeat/module/aws/s3_request/s3_request.go [hunk 2]
```diff
@@ -216,5 +216,6 @@ func createS3RequestEvents(outputs []cloudwatch.MetricDataResult, regionName str
 
 	resultMetricSetFields.Put("bucket.name", bucketName)
 	event.MetricSetFields = resultMetricSetFields
+	event.RootFields.Put("aws.s3.bucket.name", bucketName)
```
Comment (jsoriano) [inline x-pack/metricbeat/module/aws/s3_request/s3_request.go]: Is this field documented?

File: x-pack/metricbeat/module/aws/s3_daily_storage/_meta/data.json (hunk 1 above)
Comment (kaiyan-sheng) [reply x-pack/metricbeat/module/aws/s3_daily_storage/_meta/data.json]: This metricset is still in beta, can we just remove it? If not, I can mark this field as deprecated for sure!

Comment (jsoriano) [reply x-pack/metricbeat/module/aws/s3_daily_storage/_meta/data.json]: Yes, we can remove it then :slightly_smiling_face:

File: x-pack/metricbeat/module/aws/s3_request/s3_request.go (hunk 2 above)
Comment (kaiyan-sheng) [reply x-pack/metricbeat/module/aws/s3_request/s3_request.go]: I just added this field into fields.yml, thanks!

**Comment by kaiyan-sheng** [comment]
@jsoriano I double checked and found out in filebeat, `aws.s3.bucket.name` has already been reported.

**Pull request merged**